# Generated by Django 4.1.2 on 2022-10-19 00:53

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('feria', '0002_usuarios'),
    ]

    operations = [
        migrations.CreateModel(
            name='Db',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('drop_table_fastferia_saldos_field', models.CharField(blank=True, db_column='drop table fastferia.saldos;', max_length=4000, null=True)),
            ],
            options={
                'db_table': 'db',
                'managed': False,
            },
        ),
        migrations.CreateModel(
            name='SdwErrDb',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('ora_err_number_field', models.FloatField(blank=True, db_column='ora_err_number$', null=True)),
                ('ora_err_mesg_field', models.CharField(blank=True, db_column='ora_err_mesg$', max_length=2000, null=True)),
                ('ora_err_rowid_field', models.TextField(blank=True, db_column='ora_err_rowid$', null=True)),
                ('ora_err_optyp_field', models.CharField(blank=True, db_column='ora_err_optyp$', max_length=2, null=True)),
                ('ora_err_tag_field', models.CharField(blank=True, db_column='ora_err_tag$', max_length=2000, null=True)),
                ('drop_table_fastferia_saldos_field', models.CharField(blank=True, db_column='drop table fastferia.saldos;', max_length=32767, null=True)),
            ],
            options={
                'db_table': 'sdw$err$_db',
                'managed': False,
            },
        ),
    ]
